src/vm/gdbjit.cpp

```diff
@@ -2877,6 +2928,8 @@ void NotifyGdb::MethodPitched(MethodDesc* methodDescPtr)
     if (pCode == NULL)
         return;
 
+    CrstHolder _crst(&JitDescriptorCrst);
```

Comment:
Naming convention - could you please remove the leading underscore from the name?

Reply:
done